java/ql/src/Security/CWE/CWE-789/UnboundedAllocationDeserializationBad.java

```diff
@@ -0,0 +1,22 @@
+class A implements Serializable{
```

Comment:
This file is intended to be included as a snipet in the qhelp file rather than compiled on its own, so it shouldn't matter that the class name doesn't match the file name.